packages/dialtone-vue2/recipes/item_layout/contact_info/contact_info.test.js

```diff
@@ -115,6 +115,22 @@ describe('DtRecipeContactInfo Tests', () => {
       });
     });
 
+    describe('When `avatarColor` is defined', () => {
+      beforeEach(async () => {
+        await wrapper.setProps({
+          avatarColor: '000',
+        });
+        await wrapper.vm.$nextTick();
```

Comment:
This shouldn't be needed, please test if it works after removing it.

Reply:
The set of `avatarColor ` value is needed. The next part can be deleted and the test still works as expected. Thanks for the comment.